canonical/ubuntu.com
# Update /certified paths to favor pretty links

## Done

At the moment[ https://ubuntu.com/certified?q=&category=Desktop](https://ubuntu.com/certified?q=&category=Desktop) and [Certified desktops | Ubuntu](https://ubuntu.com/certified/desktops)  look different. They shouldn’t. The former (the regular search page) should redirect to the latter (the category page).

Pretty URLs should take precedence, so queries like[ https://ubuntu.com/certified?q=Nvidia&category=Desktop](https://ubuntu.com/certified?q=&category=Desktop) should be redirected to[ https://ubuntu.com/certified/desktops?q=Nvidia](https://ubuntu.com/certified/desktops?q=Nvidia) (URL should remain unchanged when more than one category is selected)

When there is more than one category selected, URL structure should remain unchanged: https://ubuntu.com/certified?q=Nvidia&category=Desktop&category=Laptops

## QA

- Check https://ubuntu-com-13051.demos.haus/certified?q=g5&category=Desktop redirects to https://ubuntu-com-13051.demos.haus/certified?q=g5&category=Desktop
- Check https://ubuntu-com-13051.demos.haus/certified?q=Nvidia&category=Desktop redirects to https://ubuntu-com-13051.demos.haus/certified/desktops?q=Nvidia
- Check https://ubuntu-com-13051.demos.haus/certified?q=Nvidia&category=Desktop&category=Laptop remains unchanged

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4665?atlOrigin=eyJpIjoiMDhiYTUyZDkwYWI0NGQxZWJkOTA1MWRlMWVjYTk3YTMiLCJwIjoiaiJ9

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
